flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/xa/JdbcExactlyOnceSinkE2eTest.java

```diff
@@ -212,7 +212,7 @@ public void testInsert() throws Exception {
                         JdbcSink.exactlyOnceSink(
                                 String.format(INSERT_TEMPLATE, INPUT_TABLE),
                                 JdbcITCase.TEST_ENTRY_JDBC_STATEMENT_BUILDER,
-                                JdbcExecutionOptions.builder().build(),
+                                JdbcExecutionOptions.builder().withMaxRetries(0).build(),
```

Comment:
Could you please mention the requirement to pass retries==0 in `JdbcSink#exactlyOnceSink` javadoc?